mediator/med_phases_history_mod.F90

```diff
@@ -1058,6 +1059,10 @@ subroutine med_phases_history_write_comp_aux(gcomp, compid, auxcomp, rc)
     integer                 :: yr,mon,day,sec    ! time units
     real(r8)                :: time_val          ! time coordinate output
     real(r8)                :: time_bnds(2)      ! time bounds output
+    !DEBUG
```

Comment:
Should this code be removed?

Reply:
@jedwards4b - yes - could you please remove it.